Features/exacte_match.feature

```diff
@@ -46,7 +50,7 @@ Abstract scenario: geen exacte match ondanks exacteMatch=true
   Dan bevat het resultaat geen objecten
 
 Abstract Scenario: combinatie van huisnummer, huisletter en/of huisnummertoevoeging komt meerdere keren voor
-  Gegeven een straat met postcode 1234AA en twee objecten met huisnummer 3
+  Gegeven een object met postcode 1234AA en twee objecten met huisnummer 3
   Als postcode=1234AA
   En huisnummer=3
   En optioneel huisnummertoevoeging
```

Comment:
Dat deze optioneel zijn, is moeilijk te bevatten omdat er altijd gegevens moeten zijn die de nummeraanduiding uniek maken. Oftewel, dit moet nader gespecificeerd worden.